Hearthstone Deck Tracker/Enums/ShowDateOnDecksOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel;
+
+namespace Hearthstone_Deck_Tracker.Enums
+{
+	public enum ShowDateOnDecksOptions
+	{
+		[LocDescription("Enum_Show_No_Date")]
+		showNoDate,
+		[LocDescription("Enum_Last_Time_Played")]
+		showLastPlayedDate,
+		[LocDescription("Enum_Last_Time_Edited")]
+		showLastEditedDate
```

Comment:
Should start with uppercase

Reply:
With renaming the enum to `DateType` this can just be `LastEdited`. (localization string should be renamed too)